lib/src/platform_client.dart

```diff
@@ -368,35 +367,31 @@ class PlatformClient {
   Future<void> uploadPhoto(int serviceRequestId, ByteBuffer photo) async {
```

Comment:
that uploadPhoto function wasn't following the usual pattern in error management.